openff/toolkit/topology/topology.py

```diff
@@ -1562,7 +1559,7 @@ def from_pdb(
         All molecules in the PDB file have the following requirements:
 
         * Polymer molecules must use the standard atom names described in the
-          `PDB Chemical Component Dictionary <https://www.wwpdb.org/data/ccd>`_
+          `PDB Chemical Component dictionary <https://www.wwpdb.org/data/ccd>`_
```

Comment:
```suggestion
          `PDB Chemical Component Dictionary <https://www.wwpdb.org/data/ccd>`_
```

I wasn't sure about the other uppercase -> lowercase changes in prose, but this one I'm relatively confident is wrong

Reply:
Ha, I'm surprised this is in the final diff. This change probably comes from me being overzealous in reverting what I think was overzealous `sed`ing ...

Reply:
Scratch that - I searched the term and saw it capitalized here: https://www.wwpdb.org/data/ccd

Reply:
"Searched" bringing up the same link in the source code 😆